app/uk/gov/hmrc/breathingspaceifproxy/config/AppConfig.scala

```diff
@@ -60,4 +60,14 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
     config.get[Seq[String]]("api.access.version-1.0.allowlistedApplicationIds")
 
   val memorandumFeatureEnabled = config.get[Boolean]("feature.flag.memorandum.enabled")
+
+  lazy val unavailableMemorandumPeriodDuration =
```

Comment:
You can use object nesting here to avoid long names and make it easier to understand. Something like:

```
object CircuitBreaker {
    private lazy val circuitBreaker = config.get[Configuration]("circuit.breaker")

    private trait CirciutBreakerConfig {
      protected def root: Configuration

      lazy val numberOfCallsToTriggerStateChange: Int = root.get[Int]("failedCallsInUnstableBeforeUnavailable")
      lazy val unavailablePeriodDuration: Int = root.get[Int]("unavailablePeriodDurationInMillis")
      lazy val unstablePeriodDuration: Int = root.get[Int]("unstablePeriodDurationInMillis")
    }

    object Memorandum extends CirciutBreakerConfig {
      protected lazy val root = circuitBreaker.get[Configuration]("memorandum")
    }

    object IF extends CirciutBreakerConfig {
      protected lazy val root = circuitBreaker.get[Configuration]("root")
    }
  }
```

One of the old configs specifies a specif config for dev, just remove this and implement in `app-config-development`